lib/helper/ExtensionElementsHelper.js

```diff
@@ -7,23 +7,20 @@ var is = require('bpmn-js/lib/util/ModelUtil').is;
 
 var ExtensionElementsHelper = {};
 
-var getExtensionElements = function(bo) {
```

Comment:
This helper inside the helper was confusing, since its name is the same as one of the exports, and it does something else. It was a one-liner anyway, and was only used in one place, so I removed it.